lib/src/polymer_base.dart

```diff
@@ -0,0 +1,292 @@
+// Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+library polymer_interop.src.js_element_proxy;
+
+import 'dart:async';
+import 'dart:html'
+    show
+        Element,
+        DocumentFragment,
+        CustomEvent,
+        Node,
+        LinkElement,
+        DocumentFragment,
+        TemplateElement;
+import 'dart:js' as js;
+import 'package:web_components/web_components.dart'
+    show CustomElementProxyMixin;
+
+/// A mixin to make it easier to interoperate with Polymer JS elements.
+///
+/// You must also mixin an implementation of the [CustomElementProxyMixin] class
+/// `web_components`. The implements statement here enforces that.
```

Comment:
nit: enforces that => hints that?

(technically it doesn't enforce it - users can satisfy the 'implements' by adding an implementation of every member declared in CustomElementProxyMixin.)